README.md

```diff
@@ -230,6 +230,10 @@ And another terminal with:
 script/ngrok
 ```
 
+Then enable features with:
```

Comment:
I'd phrase this to make it sound optional, instead of mandatory. Production is only running with 2 of 4 features enabled, so a developer might not want to enable all the features.